test/utils.test.js

```diff
@@ -0,0 +1,25 @@
+import { isDifferentActiveNode } from '../src/render/idyll-display/utils';
+const expect = require('expect');
+
+describe('isDifferentActiveNode tests', () => {
+  it('should return true with first node null', () => {
```

Comment:
Could we add (to the second test too) a slight wording update with `...return true with *just* first(second) node null`?

Reply:
Updated!